graphrag_sdk/ontology.py

```
@@ -103,12 +103,58 @@ def from_graph(graph: Graph):
+        Returns:
+            Ontology: The Ontology object constructed from the Knowledge Graph.
+        """
+        ontology = Ontology()
+
+        # Retrieve all node labels and edge types from the graph.
+        n_labels = graph.call_procedure("db.labels").result_set
+        e_types = graph.call_procedure("db.relationshipTypes").result_set
+        
+        # Extract attributes for each node label, limited by the specified sample size.
+        for lbls in n_labels:
+            l = lbls[0]
```

Ruff (0.8.2)

Ambiguous variable name: `l`

(E741)